pkg/pillar/zedcloud/wstunnelclient.go

```diff
@@ -98,7 +99,7 @@ func (t *WSTunnelClient) TestConnection(proxyURL *url.URL, localAddr net.IP) err
 
 	serverName := strings.Split(t.TunnelServerNameAndPort, ":")[0]
 	// XXX assume v1 API for now
-	tlsConfig, err := GetTlsConfig(nil, serverName, nil, false)
+	tlsConfig, err := GetTlsConfig(devNetStatus, serverName, nil, false)
```

Comment:
I don't see any code for wstunnel where it reacts to DeviceNetworkStatus to do the updateTLSProxyCerts.
We need that if we want it to be able to move to V2.

Reply:
this one calls GetTlsConfig everytime. So no need for update